docs/network-iot/hotspots-iot/data-only/balena-data-only-hotspot.mdx

```diff
@@ -15,11 +15,11 @@ import LegacyContentBanner from '@site/src/theme/LegacyContentBanner'
 
 <LegacyContentBanner />
 
-A data-only Hotspot is a type of Hotspot that participates on the Helium Network by transferring
-data from the end devices (LoRaWAN nodes). The data-only Hotspots are eligible for Network Data
-Transfer Rewards but not Proof of Coverage Rewards.
+A Data-Only Hotspot is a type of Hotspot that participates on the Helium Network by transferring
+data from the end devices (LoRaWAN nodes). The Data-Only Hotspots are eligible for Network
+Data Transfer rewards but not Proof-of-Coverage rewards.
```

Comment:
data transfer does not need to be capitalized

Reply:
It was in the original text, but I will do all lower case letters.